pkg/util/mocks/openshiftcluster.go

```diff
@@ -0,0 +1,56 @@
+package mocks
```

Comment:
I would prefer `pkg/util/mocks` to only contain generated mocks to be able to cleanup easily: just remove who dir and regenerate everything.

Reply:
somewhere under the newly created `test/` for this and the stuff below probably makes sense.